src/coreclr/tests/src/Loader/classloader/DefaultInterfaceMethods/reabstraction/reabstraction.il

```diff
@@ -4,6 +4,7 @@
 //
 
 .assembly extern mscorlib { }
+.assembly extern System.Private.CoreLib { }
```

Comment:
This should reference System.Runtime instead. Ideally, none of the tests would reference System.Private.CoreLib directly.